src/ex14_js-context/task-02.js

```diff
@@ -0,0 +1,69 @@
+function Hangman(word) { 
+    this.errors = 6; 
```

Comment:
I'd say better naming is `errorsLeft` or `lifes` because it isn't about mistakes that the user has already made